src/main/resources/default/taglib/t/lookupValue.html.pasta

```diff
@@ -9,14 +9,15 @@
 <i:arg name="showSelectionButton" type="boolean" default="true"/>
 <i:arg name="table" type="String" default="@value.getTableName()"/>
 <i:arg name="id" type="String" default="@generateId('singleselect-%s')"/>
+<i:arg name="allowCustomEntries" type="boolean" default="true"/>
 <i:arg name="class" type="String" default="" description="Lists additional CSS classes to apply to the field."/>
 
 <i:pragma name="inline" value="true"/>
 <i:pragma name="description" value="Renders a dropdown field for a LookupValue field"/>
 
 
 <t:singleSelect name="@name" label="@label" help="@help" optional="@optional" readonly="@readonly" id="@id"
-                allowCustomEntries="@value.acceptsCustomValues()" class="@class"
+                allowCustomEntries="@allowCustomEntries && value.acceptsCustomValues()" class="@class"
```

Comment:
For boolean expressions, tagliatelle knows that an expression is present, thus no @ is needed
```suggestion
                allowCustomEntries="allowCustomEntries && value.acceptsCustomValues()" class="@class"
```

Reply:
Also, shouldn't  this overwrite the value.acceptsCustomValues()? (rather than just suppressing it?)